src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -364,9 +364,13 @@ private void LogReturnedToken(AuthenticationResult result)
                 int appHashCode = AuthenticationRequestParameters.AppConfig.GetHashCode();
                 string scopes = string.Join(" ", result.Scopes);
                 Uri canonicalAuthority = new Uri(AuthenticationRequestParameters.AuthorityInfo.CanonicalAuthority);
```

Comment:
Move this inside the IF statement pls.

Reply:
These variables are used outside of the if statement so this would cause errors